Libs/Widgets/ctkConsole.cpp

```diff
@@ -541,8 +656,72 @@ void ctkConsolePrivate::updateCompleter()
     // Get the text between the current cursor position
     // and the start of the line
     QTextCursor text_cursor = this->textCursor();
-    text_cursor.setPosition(this->InteractivePosition, QTextCursor::KeepAnchor);
+    while (!text_cursor.selectedText().contains(">>>"))
```

Comment:
@MayeulChassagnard Move this as a last commit labelled `ENH: **WIP** Support completion within multi-line statement`